### ISSUE TITLE
[Bug]: login / password autocomplete are no longer triggered

### ISSUE THREAD

**Issue by fabiencharrasse** [VueTorrent/VueTorrent#1504]
### Bug description

I use Bitwarden as my password manager and since some times (weeks, months maybe, don't remember), the login and password are not autocompleted on login page using auto fill on page load : https://bitwarden.com/help/auto-fill-browser/#on-page-load

### Expectations

Login and password should be autocompleted

### How did you install VueTorrent?

Using a third-party docker image (hotio or others)

### Environment description

```plain text
https://github.com/arafatamim/linuxserver-io-mod-vuetorrent
```

### qBittorrent version

4.6.3

### VueTorrent version

2.6.0

### Agreements

- [X] The bug still occurs on the nightly build
- [X] The bug still occurs on the latest version of qBittorrent
- [X] A similar issue doesn't already exist

**Comment by fabiencharrasse** [comment]
Sorry my first message lake of precisions. I speak about the auto fill on page load : https://bitwarden.com/help/auto-fill-browser/#on-page-load
The inline auto fill work but i don't use it.

**Comment by fabiencharrasse** [comment]
I just tested the change, ids are defined on both fields but no auto-complete :/

**Comment by fabiencharrasse** [comment]
Sorry I read it wrong. I just tested, the autofill is rather long (3 seconds) and does not fill the password field. Was the password field filling up during your tests?

**Comment by Larsluph** [comment]
> Sorry I read it wrong. I just tested, the autofill is rather long (3 seconds) and does not fill the password field. Was the password field filling up during your tests?

On page load it doesn't autofill but using any kind of manual autofill method works (the extension icon and the dropdown under the field)

**Comment by fabiencharrasse** [comment]
Yes, my bug report concerns auto completion on page loading only. I tested saving the page (html) and doing auto completion tests when the page loads on the generated html file, and it works as expected. I think the problem comes from the fact that the HTML of the login page is modified when the page loads and therefore Bitwarden cannot fill the fields because they are modified at the same time as it detects them.

**Comment by Larsluph** [comment]
I already tested and autofill on loading works if you add linked fields.

**Comment by soredake** [comment]
> Yes, my bug report concerns auto completion on page loading only.

Seems I'm having the same problem, but with keepassxc-browser addon.

**Comment by fabiencharrasse** [comment]
> I already tested and autofill on loading works if you add linked fields.

Did i miss something ? I tried using password as text field and linked field type (user / password) too. No success.

![image](https://github.com/VueTorrent/VueTorrent/assets/86050782/dc0ee294-7341-404b-a98b-ee0f285ad7d6)

**Comment by Larsluph** [comment]
I don't think you're using linked fields correctly, it should look like that:

![image](https://github.com/VueTorrent/VueTorrent/assets/22910497/581247ed-b71e-4d0a-ad0d-41e7321ef12a)

**Comment by fabiencharrasse** [comment]
> I tried using password as text field and _**linked field type**_ (user / password) too

I tried all the options offered by bitwarden, including this one :

![image](https://github.com/VueTorrent/VueTorrent/assets/86050782/385186bc-9616-4b5e-b187-6be277dd2c6f)

**Comment by Larsluph** [comment]
Looking at the [source](https://github.com/bitwarden/clients/blob/main/apps/browser/src/autofill/content/autofiller.ts), they use the `DOMContentLoaded` event with a 500ms delay to autofill, when trying to reproduce I got only the username autofilling. Weird behaviour as both field should be added at the same time in the DOM...

**Comment by fabiencharrasse** [comment]
This seems to be a common issue on Bitwarden's side: https://github.com/bitwarden/clients/issues/1621
I can't seem to find a guide to help make this work.

**Comment by zcraftelite9495** [comment]
Possible fix for Password Manager compatibility for this and #2358 are addressed in PR #2361. Note that in most cases you will need to have a NGINX mapped web address in order to use some password managers that do not support localhost.

**Comment by Larsluph** [comment]
Do you have any errors that show up in the DevTools console?

**Comment by JWWolstenholme** [comment]
> If I manually click on the password field (in VueTorrent login) and then click on Bitwarden to fill it in again, it does it. I think something is off with the password field perhaps?

I've also been experiencing this since sometime recently, like within the last month or two very roughly.

I've tried changing a bunch of things in [`src/pages/Login.vue`](https://github.com/VueTorrent/VueTorrent/blob/85cfc7e97d8f1fbe1051d236b38f5afd8a4b591c/src/pages/Login.vue) in my dev environment to no avail.

Only thing of note, autofill works just fine if there is already any arbitrary value in the fields. So supposedly that narrows it down to something about the element changing when it contains a value, but I couldn't narrow down exactly what, or come up with a workaround. GIF below to demonstrate.

Bitwarden version: 2025.10.2
VueTorrent version: 85cfc7e97d8f1fbe1051d236b38f5afd8a4b591c

![Image](https://github.com/user-attachments/assets/003124b4-987b-4d45-bb2d-86702846bef5)

**Comment by Samaze123** [comment]
I don't really know VueJS but I tested anyway and I could make my password manager autofill both fields when I replaced the `v-form` by `form` ***(pure HTML)*** and the `v-text-field` ***(username and password)*** by `input`. 
I know it's not the right solution as it's not using the Vue components but it means it's because the `v-form` and `v-text-field`don't contain the right metadata or something else.
